sssd_test_framework/hosts/ldap.py

```diff
@@ -35,6 +35,14 @@ def __init__(self, *args, **kwargs) -> None:
         self.client.setdefault("id_provider", "ldap")
         self.client.setdefault("ldap_uri", f"ldap://{self.hostname}")
 
+        # Use different default for domain
```

Comment:
Nitpick: Most of the comments have a period at the end. It could be more consistent, just going with what Pavel set.